cmd/kubeadm/app/phases/certs/certs_test.go

```diff
@@ -803,3 +805,17 @@ func TestCreateCertificateFilesMethods(t *testing.T) {
 		testutil.AssertFileExists(t, tmpdir, test.expectedFiles...)
 	}
 }
+
+func parseCertAndKey(basePath string, t *testing.T) (*x509.Certificate, crypto.PrivateKey) {
```

Comment:
This function seems used only in `certlist_test.go`, so might be better to move it there